code/modules/mob/living/carbon/carbon.dm

```diff
@@ -95,7 +92,7 @@
 			visible_message(span_danger("[src] crashes into [victim][extra_speed ? " really hard" : ""], knocking them both over!"),\
 				span_userdanger("You violently crash into [victim][extra_speed ? " extra hard" : ""]!"))
 			log_combat(src, victim, "crashed into")
-
+	playsound(src,'sound/weapons/punch1.ogg',50,TRUE)
```

Comment:
Now I believe this placement here will play the sound effect every time a body is thrown, regardless of if it even hits something or just lands on the ground, because `throw_impact` is called even when something just lands on the ground

Reply:
Fuck.